index.js

```diff
@@ -1,42 +1,58 @@
 const Discord = require('discord.js');
+const {Uni} = require('./uniBot');
 const {prefix, token, giphyToken, ytToken} = require('./config');
 const client = new Discord.Client();
-const quotes = require('./quotes');
+
 const GphApiClient = require('giphy-js-sdk-core');
 giphy = GphApiClient(giphyToken);
 const fetch = require('node-fetch');
 
-client.once('ready', () => {console.log("Unicorn trotting!")});
+
+client.once('ready', () => {
+    console.log("Unicorn trotting!")
+});
 
 //!uni
 client.on('message', message => {
-    if (message.content.startsWith(`${prefix}uni`)) {
-        let member = message.mentions.members.first() || "";
-        giphy.search('gifs', {"q": ["magical", "unicorn", "animated candies", "fairyland"]})
-            .then((response) => {
-                //getting one gif
-                let totalResponses = response.data.length;
-                let responseIndex = Math.floor((Math.random() * 10) + 1) % totalResponses;
-                let finalResponse = response.data[responseIndex];
-                //getting one quote
-                let totalQuotes = quotes.length;
-                let quotesIndex = Math.floor((Math.random() * 10) + 1) % totalQuotes;
-                let quote = quotes[quotesIndex];
-
-                message.channel.send(`${quote} ${member} :unicorn:`, {
-                    files: [finalResponse.images.fixed_height.url]
-                });
-            })
-            .catch(() => {
-                message.channel.send('Hiccup...')
-            });
+    try {
+        if (message.content.startsWith(`${prefix}uni`)) Uni(message);
+        else if (message.content === 'ping') message.reply('pong');
+        else if (message.content === '!flip' || message.content === '!f') message.reply(Math.round(Math.random()) === 1 ? ' ♕ Heads!' : ' ♘ Tails!');
+        else if (message.content.startsWith('!scrim')) scrim(message)
+    } catch {
+        //message.reply('Something broke, should have written tests.');
```

Comment:
this catch block is redundant if the message is commented out

Reply:
also the initial error message was `'Hiccup...'`, and since the unit tests one is just as good, I would prefer if you'd send `one` || `the other`

Reply:
i commented it out for testing, ok